--- pytorch_lightning/core/optimizer.py
+++ pytorch_lightning/core/optimizer.py
@@ -352,8 +352,2 @@

-        if not isinstance(scheduler, LRSchedulerTypeTuple) and not is_overridden("lr_scheduler_step", model):
-            raise MisconfigurationException(
-                f"The provided lr scheduler `{scheduler.__class__.__name__}` doesn't follow PyTorch's LRScheduler"
-                " API. You should override the `LightningModule.lr_scheduler_step` hook with your own logic if"
-                " you are using a custom LR scheduler."
-            )
